drivers/net/macsec.c

```diff
@@ -940,11 +940,13 @@ static enum rx_handler_result handle_not_macsec(struct sk_buff *skb)
 
 				nskb->dev = ndev;
 				netif_rx(nskb);
+				count_rx(ndev, nskb->len);
```

Comment:
Minor: is this an intentional decision to increment counters after netif_rx()? If my memory doesn't fail me, the order is vice versa in the driver (first increment counter, then handle). Not that it should make much of a difference, but is it worth syncing the order a bit maybe

Reply:
ok, let it be in the same way everywhere